2025-05-FSC/README.md

```diff
@@ -0,0 +1,22 @@
+# Consultatie FSC release 2025
+FSC is veranderd, maar dit heeft nog steeds impact op de DK-REST profiel.
```

Comment:
Dit kan weg denk ik? - ook is het goed als de consultatie tekst hier zelfstandig te lezen is (dus met context)